script/foreman-debug

```diff
@@ -19,6 +19,8 @@ This program can be used on Foreman instances, Smart Proxy instances or
 backend services separately.
 
 OPTIONS:
+  -E      Detailed output incl. host names or domains
+          (THIS OPTION ADDS MORE SENSITIVE DATA IN THE REPORT)
```

Comment:
Do you have a particular example how the flag should look like?

I do want to export only particular items via SQL. Your idea to provide full database dump is good, but I want to implement this in a separate flag, because the dump can very quickly run out of size bounds.